frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -130,3 +130,81 @@ Feature: Create the pipeline from builder page
               And user clicks Create button on Pipeline Builder page
              Then user will be redirected to Pipeline Details page with header name "pipeline-workspace"
               And user will see workspace mentioned as "git-opt (optional)" in the Workspaces section of Pipeline Details page
+
+
+        @regression, @manual
+        Scenario: Add finally task node
+            Given user is at Pipeline Builder page
+             When user navigates to task section
+              And user clicks on Add finally task
+              And user clicks on Select task
+              And user select "git-cli"
+              And user clicks on Add finally task again
+              And user clicks on Select task
+              And user selects "git-clone"
+             Then user sees Add finally task bubble is attached to the Select Task
+              And user sees "git-cli" and "git-clone" tasks in parallel
+              And user sees "Add finally task" option below "git-clone" task
+
+
+        @regression
+        Scenario: Create a pipeline with finally task node
+            Given user is at Pipeline Builder page
+             When user enters pipeline name
+              And user selects the first task as "git-clone"
+              And user clicks on Add finally task
+              And user clicks on Select task
+              And user select "git-clone"
+              And user clicks on Add finally task again
+              And user clicks on Select task
+              And user selects "openshift-client"
+              And user clicks on Add finally task again
+              And user clicks on Select task
+              And user selects "kn"
+              And user fills task details
+              And user clicks on create
+             Then user sees finally tasks "git-clone", "openshift-client" and "kn" mentioned under "" section in the Pipeline details page
+
+
+        @regression, @manual
+        Scenario: When expression in the Pipeline Builder
+            Given user is at Pipeline Builder page
+              And user has chain of 4 tasks created in series
+             When user clicks on third task
+              And user navigates to When Expressions section
+              And user clicks on Add When Expressions
+             Then user sees a diamond shaped structure appear in front of third task
+              And user sees "Input", "Operator" and "Value" fields in When Expressions section
```

Comment:
Any interest in checking if `Operator` has only the two values? `in` and `notin`

Reply:
Added the checks